### DISCUSSION TITLE
能否禁止combos的缩放，只缩放里面的节点

### DISCUSSION THREAD

**Discussion by hbsjmsjwj** [antvis/G6#3715] [Q&A]
![to](https://user-images.githubusercontent.com/45258156/172102896-2844c86a-5221-495b-ac12-571f608ce646.png)
能否滚轮缩放，只缩放里面的节点，外层的combos“客户端”，“流量网关”框框不让他缩放

**Comment by Yanyan-Wang** [comment]
应该暂时不会考虑支持
